templates/rhpam73-prod-immutable-monitor.yaml

```diff
@@ -42,10 +42,14 @@ parameters:
   name: APPLICATION_NAME
   value: myapp
   required: true
+- displayName: Maven mirror URL
+  description: Maven mirror to use for S2I builds
+  name: MAVEN_MIRROR_URL
```

Comment:
What would this template use a Maven mirror for? It does not build or use any KJARs?